### ISSUE TITLE
[Bug]: Resourceful Creepers spawning in light levels above 0

### ISSUE THREAD

**Issue by jasoncollege24** [FTBTeam/FTB-Modpack-Issues#1001]
### Modpack

FTB Plexiglass Mountain

### Modpack version

1.0.2

### Describe the bug

Saw some resourceful creepers in a well lit stairwell more than once, when no nearby areas were dark. Also seen them spawn in a few other well lit areas.

### Steps to reproduce

1. Light up a large area, where resourceful creepers should spawn, so that no blocks have a light level of 0
2. ensure that the test area has blocks with light levels between 1, and 7
3. move to the required spawn distance
4. wait
5. watch a resourceful creeper spawn

### Expected behaviour

No Resourceful Creepers should spawn in light levels above 0

### Screenshots

![Screenshot (59)](https://user-images.githubusercontent.com/22086680/175063078-193e66a1-83e5-4d68-9059-7a0c4048c17f.png)

### Additional information

This is on a server, located on my home LAN, running ubuntu 20.04. All players play in survival, and server side configs are all defaults, except RAM, which was allocated up to 6GB.

The included screenshot shows a mob that spawned moments before it was taken, obviously where the light level is not 0.

There are no related logs to show, so none were included.

**Comment by Raidobw2** [comment]
I actually thought I'd use InControl to fix it, but turns out it has its quirks.

I ended up going with a hybrid solution, the overworld spawns are handled by the mod and the nether and end are handled by InControl since normally creepers don't spawn there.

This will fix the light issue in the Overworld only.
The Nether and End I could have put a fix for it, though only the End kinda makes sense, so I didn't change anything on that yet. (There's a lot of light in the Nether = you wouldn't as much spawns)

What I did do and what this also done as a good side effect, Resource Creepers will be a lot more common in the Overworld.
They can also spawn on the surface at night now. I also buffed Nether and End rates.

I think this should be a more fun experience, this will be in the 1.1.1 version of the pack (which I'm hoping is going to be a lot more stable).

Thanks for the reports!
